bigchaindb/commands/utils.py

```diff
@@ -198,6 +198,7 @@ def mongodb_host(host):
                               '(use "-" for stdout)')
 
 base_parser.add_argument('-l', '--log-level',
+                         type=lambda l: l.upper(),  # case insensitive conversion
```

Comment:
Why not:
```python
                         type=str.upper,  # convert to uppercase for comparison to choices
```
The relevant docs about `type=` in argparse: https://docs.python.org/3.5/library/argparse.html#type 

More discussion: https://stackoverflow.com/questions/27616778/case-insensitive-argparse-choices